sensor_board/common/pinav_parser.h

```diff
@@ -0,0 +1,90 @@
+#ifndef _PINAV_PARSER_H_
+#define _PINAV_PARSER_H_
+
+#include <stdint.h>
+
+// All flight code will be in C, but the test library is technically C++.
+// This means that all flight code must declare itself to have C linkage when
+// compiled in C++ mode.
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Status enum returned by parse function
+typedef enum pinav_parser_status	{	OK, 
+										NULL_OUTPUT_PTR,
+										NULL_SENTENCE_PTR,
+										UNRECOGNIZED_SENTENCE_TYPE,
+										IMPROPER_SENTENCE_LENGTH,
+										SENTENCE_FORMAT_ERROR,
+										UNKNOWN_ERROR // @TODO: remove once I'm sure this never happens
+									} pinav_parser_status_t;
```

Comment:
These should probably be prefixed by something (e.g. `PINAV_PASE_` or `PN_PASE_`). Should help reduce cases where we accidentally use a constant from the wrong enumeration. The `NULL_OUTPUT_PTR` and `NULL_SENTENCE_PTR` seem especially susceptible to this.